Libs/DICOM/Widgets/ctkDICOMBrowser.h

```diff
@@ -107,14 +107,14 @@ class CTK_DICOM_WIDGETS_EXPORT ctkDICOMBrowser : public QWidget
   /// Option to show or not import summary dialog.
   /// Since the summary dialog is modal, we give the option of disabling it for batch modes or testing.
   void setDisplayImportSummary(bool);
-  bool displayImportSummary();
+  bool displayImportSummary()const;
   /// Option to show dialog to confirm removal from the database (Remove action). Off by default.
   void setConfirmRemove(bool);
-  bool confirmRemove();
+  bool confirmRemove()const;
   /// Option to determine whether the new database folder is automatically created or set by the user in a popup.
   /// Automatically created folder will be ../[CurrentDatabaseFolderName]-[NewSchemaVersion]. Off by default.
```

Comment:
it seems that it is On by default now

Reply:
Right. If we have a decision on the default I'll update it here too.